.github/workflows/ci-docs.yaml

```diff
@@ -26,9 +26,11 @@ jobs:
           DEST_REF: refs/heads/preview/${{ github.event.pull_request.number }}
         run: |
           if gh api --silent "/repos/Mergifyio/docs/git/${DEST_REF}"; then
-            gh api -X POST "/repos/Mergifyio/docs/git" -f "sha=${HEAD_SHA}" -f "refs=$DEST_REF"
-          else
+            echo "Updating $DEST_REF"
```

Comment:
Should this be in another PR?

Reply:
@sileht This one is for you I think

Reply:
We need to rebase the branch to fix that one, because this change is already in main.

Reply:
@AlexandreGaubert feel free to rebase the branch and force-push when you fix the last @jd  comment.